H2TauTau/python/proto/analyzers/NJetsAnalyzer.py

```diff
@@ -101,9 +103,17 @@ def process(self, event):
                 sumpt += math.sqrt(mom.x[0]**2 + mom.x[1]**2)
                 outgoing.append(pdg)
 
+            if status==1 and abs(pdg) in [11, 13, 15]:
```

Comment:
If we'll ever need this for W+jets, it would be good to add neutrinos here as well